openmc/universe.py

```diff
@@ -395,6 +399,11 @@ def plot(self, origin=None, width=None, pixels=(200, 200),
                 y_width = bb_width['xyz'.index(basis[1])]
                 width = (x_width, y_width)
 
+        if isinstance(pixels, int):
+            aspect_ratio = width[0] / width[1]
+            pixels_y = np.sqrt(pixels / aspect_ratio)
```

Comment:
When operating on scalars, use `math.sqrt` instead of `np.sqrt`

Reply:
Thanks Paul good to know.